src/notifier/index.js

```diff
@@ -4,6 +4,7 @@ dotenv.config();
 
 import sendInBlue from 'sib-api-v3-sdk';
 
+const BASE_URL = "github.com/ambanum/CGUs-data/commit/";
```

Comment:
What about moving this in the email template instead?

Reply:
Done.

Reply:
I'm not sure… it's tempting but in the end that still means duplication, but in a place even farther away. At the same time, by the time we change this URL we'll probably change other UI elements.

I like the idea of this being in the config as it would make it easier for some actor to fork and redirect to their own database